foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/data/LocationUriInput.java

```diff
@@ -0,0 +1,39 @@
+package com.boozallen.aissemble.configuration.data;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Configuration::Store
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class LocationUriInput {
```

Comment:
Q: What are we doing with this? Looks like init reads directly from system environment.

Reply:
Ah my question was answered. I didn't realize we were implementing a `/load` method.  I think we probably want to avoid taking the URIs as input.  I'd expect `/load` just to load with the preconfigured URL.  Allowing arbitrary URIs to be passed in feels like it introduces a lot of security issues that would have to be carefully managed without providing a whole lot of value.